genai-perf/genai_perf/llm_inputs/llm_inputs.py

```diff
@@ -783,41 +771,6 @@ def _convert_generic_json_to_output_format(
 
         return output_json
 
-    @classmethod
-    def _convert_generic_json_to_openai_chat_completions_format(
```

Comment:
With the removal of `_determine_json_feature_roles`, this method can be just merged with `_populate_openai_chat_completions_output_json`